src/pages/PlaylistManagementPage/PlaylistManagementPage.js

```diff
@@ -32,23 +33,45 @@ const PlaylistManagementPage = (props) => {
   }, [])
 
   const playlistClickedHandler = (playlistId) => {
+    const playlists = [...collaborativePlaylists]
+    const playlist = playlists.find((playlist) => playlist.id === playlistId)
+
+    if(playlist.selected) {
+      axios
+        .delete(
+          `${process.env.REACT_APP_API_BASE_URL}/playlist/${playlistId}`,
+          { withCredentials: true },
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
+          setCollaborativePlaylists(() => playlists)
+        })
+        .catch((err) => console.error('Something went wrong'))
+    } else {
+      axios
+        .post(
+          `${process.env.REACT_APP_API_BASE_URL}/playlist/${playlistId}`,
+          { withCredentials: true },
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
+          setCollaborativePlaylists(() => playlists)
+        })
+        .catch((err) => console.error('Something went wrong'))
+    }
+  }
+
+  const forceReorder = () => {
```

Comment:
Can we rename this method to forceReorderClickedHandler? Just to follow the convention to specify handler methods

Reply:
Sure. I did not notice the convention. That makes sense.

Reply:
Done.